src/main/java/techreborn/items/armor/BatpackItem.java

```diff
@@ -104,4 +137,15 @@ public boolean isItemBarVisible(ItemStack stack) {
 	public int getItemBarStep(ItemStack stack) {
 		return ItemUtils.getPowerForDurabilityBar(stack);
 	}
+
+	private void registerTrinket() {
+		// Ensures that trinkets are only registered if the mod is installed
+		try {
+			Class.forName( "dev.emi.trinkets.api.TrinketsApi" );
```

Comment:
Instead use ModLoader.isLoaded(String) or the other method. Loaded is later I think and present or something sooner. Should take the sooner one

Reply:
I found another check for `if the mod existed` in the code base and just used that.